### PR TITLE
Add compose version of `OpenDeviceSettingsDialog`

### PR DESCRIPTION
**Pull request by esensar**

### CODE REVIEW

**Comment by FunkyMuse** [comment]
ahhh, i was working on this

**Comment by FunkyMuse** [comment]
also you're missing a preview for the Dialog

**Comment by esensar** [comment]
> ahhh, i was working on this

Sorry about that :/
I figured it is a small one so I just updated it quickly.

File: commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/OpenDeviceSettingsDialog.kt [hunk 1]
```diff
@@ -72,3 +75,14 @@ fun OpenDeviceSettingsAlertDialog(
         },
     )
 }
+
+@Composable
+@MyDevices
+fun OpenDeviceSettingsAlertDialogPreview() {
```
Comment (FunkyMuse) [inline commons/src/main/kotlin/com/simplemobiletools/commons/dialogs/OpenDeviceSettingsDialog.kt:81]: Previews should br private fun

**Review by FunkyMuse — APPROVED** [review]

**Comment by tibbi** [comment]
accepted, but add new dialogs in TestDialogActivity at the bottom please, not randomly in the middle...

**Pull request merged**